region/crateria/east.json

```diff
@@ -1734,6 +1784,137 @@
+                    "h_canUseMorphBombs",
+                    "SpaceJump",
+                    {"or": [
+                      "h_canCeilingBombJump",
+                      "canBePatient"
+                    ]}
+                  ],
+                  "note": "The crumble block is the leftmost flat ceiling tile."
+                },
+                {
+                  "name": "G-mode Overload PLMs by Bombing Crumble Block",
+                  "notable": false,
+                  "requires": [
+                    "h_canArtificialMorphIBJ",
+                    "h_canArtificialMorphCeilingBombJump"
+                  ],
+                  "note": "Start the IBJ on the second downward tile from the door. The crumble block is the leftmost flat ceiling tile.",
+                  "devNote": "Without canCeilingBombJump, this would take more than 10 minutes."
+                },
+                {
+                  "name": "West Ocean G-mode Overload PLMs by PBing Missile Item Softlock Risk",
+                  "notable": false,
+                  "requires": [
+                    {"itemNotCollectedAtNode": 10},
+                    "canRiskPermanentLossOfAccess",
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "Morph",
+                      {"and": [
+                        "SpringBall",
+                        {"or": [
+                          "HiJump",
+                          {"ammo": { "type": "PowerBomb", "count": 1 }},
+                          "Bombs"
+                        ]}
+                      ]}
+                    ]},
+                    {"ammo": { "type": "PowerBomb", "count": 1 }}
+                  ],
+                  "note": [
+                    "To overload the PLMs, place a PB precisely to the right of the bottom of the second overhang above the door to the Moat.",
+                    "This is at the max jump height without HiJump. Placing the PB higher or lower will not overload the PLMs without many PBs."
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "from": 16,
+          "to": [
+            {
+              "id": 14,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      {"and": [
+                        "SpaceJump",
+                        "Morph"
+                      ]},
+                      "h_canArtificialMorphIBJ",
+                      {"and": [
+                        "canTrickyDashJump",
+                        "canSpringBallJumpMidAir",
+                        "HiJump"
+                      ]}
+                    ]}
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "from": 17,
+          "to": [
+            {
+              "id": 10,
```

Comment:
17 would have to go to 14.  And I think you have to end GMode at that point.

Reply:
I put this from 17->10 and then return from 10->17. 10->14 will either require that you didn't come from below or that you `canRiskPermanentLossOfAccess`. All of the strats from 17->10 require morph and assume you go to 14, cancel gmode, and grab the item. I think this is the most clear way to do it in order to make strats from 14 be more clear.

Reply:
10 can't return to 17, GMode will have ended.

Reply:
oh right. looks like its going to 13